content/kubernetes/re-databases/set-up-ingress-controller.md

```diff
@@ -88,9 +88,12 @@ Install one of the supported ingress controllers:
         http:
           paths:
           - path: /
+            pathType: Prefix
             backend:
-              serviceName: <db-name>
-              servicePort: <db-port>
+              service:
+                name: <db-name>
+                port:
+                  number: <db-port>
     ```  
 
     For HAProxy, insert the following into the `annotations` section:  
```

Comment:
Here below, we instruct the user to set the `kubernetes.io/ingress.class` annotation, which too is deprecated since 1.19 or so.  Instead, the user should set the `.spec.ingressClassName` field with a similar `haproxy`/`nginx` value.